### ISSUE TITLE
I am trying the CaffeNet demo on my phone,but nothing happens after taking photos. 

### ISSUE THREAD

**Issue by wuhuo714** [ENCP/CNNdroid#19]
Hi,
   I am trying your demo on   https://github.com/ENCP/CNNdroid/tree/master/Demo%20Android%20Applications/CaffeNet
   but after I take photos,the application only asks me to take photos again.It doesn't show the result as the Food101 Demo Video.mp4 has.
  Do you have any ideas?
  Thanks a lot.

**Comment by latifisalar** [comment]
Hi,
Have you installed using the provided APK or through the Android Studio project?
Can you provide us the logcat of your application run in the Android Studio, so we can have a better idea what can be the problem?
Thanks

**Comment by wuhuo714** [comment]
Yes,I have followed the  Installation Guide.txt and installed using the provided APK. 
I run the application in Android Studio,
when taking photos:
04-20 02:12:35.570 1259-17933/? V/EmulatedCamera_CallbackNotifier: **** Currently enabled messages:
04-20 02:12:35.571 1259-17933/? V/EmulatedCamera_CallbackNotifier:     CAMERA_MSG_ERROR
04-20 02:12:35.571 1259-17933/? V/EmulatedCamera_CallbackNotifier:     CAMERA_MSG_FOCUS
04-20 02:12:35.571 1259-17933/? V/EmulatedCamera_CallbackNotifier:     CAMERA_MSG_ZOOM
04-20 02:12:35.571 1259-17933/? V/EmulatedCamera_CallbackNotifier:     CAMERA_MSG_POSTVIEW_FRAME
04-20 02:12:35.571 1259-17933/? V/EmulatedCamera_CallbackNotifier:     CAMERA_MSG_RAW_IMAGE_NOTIFY
04-20 02:12:35.571 1259-17933/? V/EmulatedCamera_CallbackNotifier:     CAMERA_MSG_PREVIEW_METADATA
04-20 02:12:35.571 3279-3279/com.android.camera V/camera: mPictureDisplayedToJpegCallbackTime = 2ms
04-20 02:12:35.574 1272-17936/? I/OMXClient: MuxOMX ctor
04-20 02:12:35.574 1268-1268/? I/OMXMaster: makeComponentInstance(OMX.google.vorbis.decoder) in mediacodec process
04-20 02:12:35.575 3279-3279/com.android.camera D/CameraStorage: External storage state=mounted
04-20 02:12:35.576 3279-3279/com.android.camera V/camera: mJpegCallbackFinishTime = 5ms
04-20 02:12:35.586 1268-10405/? E/OMXNodeInstance: setConfig(4f4002c:google.vorbis.decoder, ConfigPriority(0x6f800002)) ERROR: Undefined(0x80001001)
04-20 02:12:35.586 1272-17936/? I/ACodec: codec does not support config priority (err -2147483648)
04-20 02:12:35.586 1272-17936/? I/MediaCodec: MediaCodec will operate in async mode
04-20 02:12:35.596 1272-17934/? D/AudioTrack: Client defaulted notificationFrames to 11264 for frameCount 22528
04-20 02:12:35.600 1272-17774/? D/NuPlayerDriver: notifyListener_l(0xb3928ea0), (6, 0, 0), loop setting(0, 0)
04-20 02:12:35.618 1272-17935/? I/NuPlayerDecoder: [audio] saw output EOS
04-20 02:12:35.619 1258-1305/? D/AudioFlinger: mixer(0xb3783580) throttle end: throttle time(42)
04-20 02:12:35.673 1258-1305/? D/AudioFlinger: mixer(0xb3783580) throttle end: throttle time(31)
04-20 02:12:35.739 1272-17774/? D/NuPlayerDriver: notifyListener_l(0xb3928ea0), (2, 0, 0), loop setting(0, 0)


 then I click the tick botton,it show
 [ 04-20 02:14:29.663  1259: 1579 D/         ]
                                                 HostConnection::get() New Host Connection established 0xb0a7c2c0, tid 1579
04-20 02:14:29.664 1259-1579/? D/gralloc_ranchu: gralloc_unregister_buffer: exiting HostConnection (is buffer-handling thread)
04-20 02:14:29.664 1259-1579/? V/EmulatedCamera_Preview: setPreviewWindow: current: 0xb0a690b0 -> new: 0x0
04-20 02:14:29.664 1259-1579/? I/CameraFlashlight: Destroying camera 1
04-20 02:14:29.664 1259-1579/? V/EmulatedCamera_Camera: closeCamera
04-20 02:14:29.664 1259-1579/? V/EmulatedCamera_Camera: doStopPreview
04-20 02:14:29.665 1259-1579/? I/CameraService: disconnect: Disconnected client for camera 1 for PID 3279
04-20 02:14:29.666 1272-1296/? D/NuPlayerDriver: reset(0xb3928ea0) at state 6
04-20 02:14:29.666 1272-1296/? D/NuPlayerDriver: notifyListener_l(0xb3928ea0), (8, 0, 0), loop setting(0, 0)
04-20 02:14:29.669 1272-17774/? W/AMessage: failed to post message as target looper for handler 0 is gone.
04-20 02:14:29.670 1272-17774/? D/NuPlayerDriver: notifyResetComplete(0xb3928ea0)
04-20 02:14:29.671 1272-1296/? D/NuPlayerDriver: reset(0xb3929320) at state 4
04-20 02:14:29.671 1272-1296/? D/NuPlayerDriver: notifyListener_l(0xb3929320), (8, 0, 0), loop setting(0, 0)
04-20 02:14:29.671 1272-17776/? D/NuPlayerDriver: notifyResetComplete(0xb3929320)
04-20 02:14:29.672 1272-1272/? D/NuPlayerDriver: reset(0xb3928de0) at state 4
04-20 02:14:29.672 1272-1272/? D/NuPlayerDriver: notifyListener_l(0xb3928de0), (8, 0, 0), loop setting(0, 0)
04-20 02:14:29.672 1272-17778/? D/NuPlayerDriver: notifyResetComplete(0xb3928de0)
04-20 02:14:29.682 1484-1497/system_process I/ActivityManager: START u0 {act=com.android.camera.action.CROP typ=image/* cmp=com.android.gallery/com.android.camera.CropImage (has extras)} from uid 10074 on display 0
04-20 02:14:29.712 1247-1379/? D/gralloc_ranchu: gralloc_alloc: format 1 and usage 0x900 imply creation of host color buffer
04-20 02:14:29.755 1247-1379/? D/gralloc_ranchu: gralloc_alloc: format 1 and usage 0x900 imply creation of host color buffer
04-20 02:14:29.793 1247-1379/? D/gralloc_ranchu: gralloc_alloc: format 1 and usage 0x900 imply creation of host color buffer
04-20 02:14:29.844 29642-29666/com.library.example.caffenet E/EGL_emulation: tid 29666: eglSurfaceAttrib(1174): error 0x3009 (EGL_BAD_MATCH)
#04-20 02:14:29.844 29642-29666/com.library.example.caffenet W/OpenGLRenderer: Failed to set EGL_SWAP_BEHAVIOR on surface 0x99cf4b40, error=EGL_BAD_MATCH

Thanks a lot!
